big_o/big_o.py

```diff
@@ -173,3 +173,52 @@ def big_o(func, data_generator,
         fitted['times'] = time
 
     return best, fitted
+
+def print_report(best, others):
+    """ Prints the detailed summary of the calculated reports
+
+    Input:
+    ------
+
+    best -- Object representing the complexity class that best fits
+            the measured execution times.
+
+    others -- A dictionary of fitted complexity classes to the residuals
+
+    Output:
+    -------
+
+    None
+
+    """
+
+    def print_line(c_rank, name, coef, residual):
+        print('{:>5}  |  {:12}  |  {:35}  |  {:.2G}'.format(
+            c_rank, name, coef, residual))
```

Comment:
Please no nested functions. If it's really necessary to have sub-functions to achieve this task, then rather as private functions at the scope of the module (e.g., `reports._parse_data_for_big_o_report`)